### PR TITLE
AWS backup and restore tech notes

### PR DESCRIPTION
**Pull request by prat98**

### CODE REVIEW

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[id="tech-note-aws-backup-restore"]
+
+= Backup and Restore
+
+The aws_rds_db_snapshot_arn optional parameter which allowed customers to specify a specific AWS RDS Snapshot to use for restoring their deployment is no longer supported. The restore process no longer uses AWS RDS Snapshots to restore a deployment and instaed relies on the backup files generated by the Ansible on Clouds Ops Container backup playbook. The Ansible on Clouds Ops Container restore playbook will not use any provided RDS snapshot passed as a parameter.
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:5]: aws_rds_snapshot_arn enclose in backticks

and instead relies

restore playbook does not use any

**Review by ianf77 — COMMENTED** [review]
few things to fix

**Comment by nathanweatherly** [comment]
Hey @ianf77 !

In your opinion, would this have been a good candidate for a HackMD file?

It's a relatively small addition, but it (a) required adding an assembly line and (b) had some special formatting. 

Thanks!

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+. The `aws_rds_db_snapshot_arn` optional parameter which allowed customers to specify a specific AWS RDS Snapshot to use for restoring their deployment is no longer supported. The restore process no longer uses AWS RDS Snapshots to restore a deployment and instead relies on the backup files generated by the Ansible on Clouds Ops Container backup playbook. The Ansible on Clouds Ops Container restore playbook does not use any provided RDS snapshot passed as a parameter.
+
+. A restored {AAPonAWS} deployment will not maintain all secrets from the original {AAPonAWS} deployment. If using your own certificates, they will need to be reset after a restore.
+
++
+.Secrets that will be maintained:
+
+* `<stack_name>-aap-admin-secret`
+* `<stack_name>-aap-secret-key`
++
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:14]: remove lines 14-17 and remove the + on line 18.

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
+[id="tech-note-aws-backup-restore"]
+
+= Backup and Restore
+
+. The `aws_rds_db_snapshot_arn` optional parameter which allowed customers to specify a specific AWS RDS Snapshot to use for restoring their deployment is no longer supported. The restore process no longer uses AWS RDS Snapshots to restore a deployment and instead relies on the backup files generated by the Ansible on Clouds Ops Container backup playbook. The Ansible on Clouds Ops Container restore playbook does not use any provided RDS snapshot passed as a parameter.
+
+. A restored {AAPonAWS} deployment will not maintain all secrets from the original {AAPonAWS} deployment. If using your own certificates, they will need to be reset after a restore.
+
++
+.Secrets that will be maintained:
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:10]: Secrets that are maintained

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+* `<stack_name>-aap-admin-secret`
+* `<stack_name>-aap-secret-key`
++
+
+//
+
++
+.Secrets that will NOT be maintained:
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:19]: Secrets that are not maintained

**Review by ianf77 — COMMENTED** [review]
Some comments

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
+[id="tech-note-aws-backup-restore"]
+
+= Backup and Restore
+
+. The `aws_rds_db_snapshot_arn` optional parameter which allowed customers to specify a specific AWS RDS Snapshot to use for restoring their deployment is no longer supported. The restore process no longer uses AWS RDS Snapshots to restore a deployment and instead relies on the backup files generated by the Ansible on Clouds Ops Container backup playbook. The Ansible on Clouds Ops Container restore playbook does not use any provided RDS snapshot passed as a parameter.
+
+. A restored {AAPonAWS} deployment will not maintain all secrets from the original {AAPonAWS} deployment. If using your own certificates, they will need to be reset after a restore.
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:7]: does not maintain...

certificates, they must be...

**Review by ianf77 — COMMENTED** [review]
another comment

**Comment by ianf77** [comment]
I don't think this would have made a HackMD file. Some of the formatting was unnecessary.

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+. A restored {AAPonAWS} deployment does not maintain all secrets from the original {AAPonAWS} deployment. If using your own certificates, they must be reset after a restore.
+
++
+.Secrets that are maintained:
+
+* `<stack_name>-aap-admin-secret`
+* `<stack_name>-aap-secret-key`
+
++
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:15]: Delete line 15

**Review by ianf77 — COMMENTED** [review]
Two minor fixes

File: stories/topics/tech-note-aws-backup-restore.adoc (hunk 6 above)
Comment (prat98) [reply stories/topics/tech-note-aws-backup-restore.adoc:15]: same with here I saw that removing the + removes the indentation

File: stories/topics/tech-note-aws-backup-restore.adoc [hunk 7]
```diff
@@ -21,3 +21,7 @@
 * `<stack_name>-container_auth_public_key_pem`
 * `<stack_name>-pulp_cert`
 * `<stack_name>-pulp_key`
+
+. Cross-region backup and restore operations are not supported in this release. Meaning a backed up deployment in one region can only be restored in to that same region.
```
Comment (ianf77) [inline stories/topics/tech-note-aws-backup-restore.adoc:25]: This means that a backed up...

**Review by ianf77 — COMMENTED** [review]
One comment

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**